tidy3d/web/s3utils.py

```diff
@@ -186,23 +186,26 @@ def _call_back(bytes_in_chunk):
             )
 
 
-def download_file(resource_id: str, remote_filename: str, to_file: str = None):
+def download_file(resource_id: str, remote_filename: str, to_file: str = None, show_progress=True):
     """
     download file from S3
     @param resource_id: the resource id, e.g. task id
     @param remote_filename: the remote file name on S3
```

Comment:
@param remote_filename: the remote file path on S3 which is relative to resource context root path. 
For example, the resource context root path will be s3://tidy3d_bucket/{resource_id}, the passed remote_filename
will be results/user.log. Then the actual s3 path will be s3://tidy3d_bucket/{resource_id}/results/user.log

Reply:
I updated the docstrings, thanks.